2-mandatory/3-stocks.js

```diff
@@ -64,10 +79,33 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
-}
-
-
+    let highestArray = [];
+    let g = 0;
+    for(elementS of closingPricesForAllStocks){
+        let highestPF = 0;
+        let highestP = elementS[0];
+        let highestN = STOCKS[g].toUpperCase();
+        for(i = 0; i <= elementS.length; i++){
+            if (highestP < elementS[i]){
+                highestP = elementS[i];
+            }else highestP = highestP;
+        }
```

Comment:
Hi Rihanna, good that you checked all reviews and got back to me with questions
What I meant was adding `highestPF = Math.round(highestP * 100) / 100` before the line you pushed values into the array and obviously replace it with `highestP` in return

Reply:
Thank you so much.
I did exercises and corrected my problems.

Reply:
Ok cool! 
I'm leaving this comment just for clarify as I thought it was a bit confusing that why you should keep toFixed() in your code after using Math.round() though! This just helps to get rid of two decimal issue you had in the test and pass it!

- Math.round() returns the nearest integer to the given number. For example, Math.round(3.5) returns 4, and Math.round(3.4) returns 3.
- toFixed() returns a string representation of the number with the specified number of decimal places. For example, 3.14159.toFixed(2) returns "3.14".